server/service/linux_mdm.go

```diff
@@ -42,3 +43,20 @@ func (svc *Service) LinuxHostDiskEncryptionStatus(ctx context.Context, host flee
 		Status: &verified,
 	}, nil
 }
+
+func (svc *Service) GetLinuxDiskEncryptionSummary(ctx context.Context, teamId *uint) (*fleet.MDMLinuxDiskEncryptionSummary, error) {
+	if err := svc.authz.Authorize(ctx, fleet.MDMConfigProfileAuthz{TeamID: teamId}, fleet.ActionRead); err != nil {
+		return nil, ctxerr.Wrap(ctx, err)
+	}
+
+	if svc.config.Server.PrivateKey == "" {
```

Comment:
No reason to include this in the endpoint to fail it. We don't want to block the config profiles endpoint for customers who only have Windows MDM enabled (which doesn't require the private key).